purchase_security/security/security.xml

```diff
@@ -30,6 +38,15 @@
             >['|',('user_id','=',user.id),('user_id','=',False)]</field>
             <field name="groups" eval="[(4,ref('group_purchase_own_orders'))]" />
         </record>
+        <record model="ir.rule" id="purchase_order_group_purchase_order_group_orders">
+            <field name="name">View purchase orders (group responsible)</field>
+            <field name="model_id" ref="purchase.model_purchase_order" />
+            <field
+                name="domain_force"
+            >['|', '|', ('user_id', '=', user.id), ('user_id', '=', False), '|', ('team_id', '=', user.purchase_team_id.id),('team_id', '=', False)]
```

Comment:
Why adding the user part? That one is already covered by the inferior record rule, isn't it?

Reply:
Thanks for all the comments. You are right, I wasn't fully understanding the difference between this rules, thats wh i was checking the user twice. I made the suggested changes.